pkg/performanceprofile/profilecreator/profilecreator.go

```diff
@@ -242,6 +242,31 @@ func (ghwHandler GHWHandler) CPU() (*cpu.Info, error) {
 	return ghw.CPU(ghwHandler.snapShotOptions)
 }
 
+func (ghwHandler GHWHandler) SortedCPU() (*cpu.Info, error) {
```

Comment:
this feature should be proposed to GHW - we should file an issue at least.
Additionally, I cant' help but keep wondering if instead of mangling `cpu.Info` we shoudl just compute here the representation we need, which seems a simple `[]int`.
IOW, instead of

1. computing a new `cpuInfo` with the sorted values
2. iterate over the mangled `cpuInfo` to extract the cpuid slice
3. use the cpuid slice

just compute the cpuid slice here and use it directly

Reply:
Sorry but I thin I do not fully understand your comment. 

we need all the socket -> core -> logical processor structure if we want to use that information to selectively disable some processors.

Reply:
right, we need to keep some hierarchy. I was focusing a too narrow use case.

Reply:
so ok, also considering we have already `SortedTopology` let's keep this as is and let's address possible cleanups/ghw extensions later